config.example.js

```diff
@@ -3,7 +3,12 @@ var config = {
     // Lenguage for the mirror
     language : "en-US",
     
-    // Keyword Spotting (Hotword Detection)
+    // PIR Detection
+    pir-hdmi : {
+        pirPin : 26, //Default pirPin is GPIO pin 26.
+		ScreenTimeOut : 5.0 //Default timeout is 5 minutes must be a float number.
+    },
+	// Keyword Spotting (Hotword Detection)
```

Comment:
the keyword comment i'm not sure how to fix... but i've updated the indentation...